cgmes/cgmes-conversion/src/test/java/com/powsybl/cgmes/conversion/test/CondenserImportExportTest.java

```diff
@@ -35,6 +35,19 @@ void cgmes2condenserConversionTest() {
         Generator g = network.getGenerators().iterator().next();
         assertEquals(0, g.getMinP());
         assertEquals(0, g.getMaxP());
+        assertTrue(g.isCondenser());
+    }
+
+    @Test
+    void cgmes2condenserExportTest() throws IOException {
+        Network network = Network.read("condenser2_EQ.xml", getClass().getResourceAsStream("/issues/condenser2_EQ.xml"));
+        String basename = "condenser2";
+        network.write("CGMES", null, tmpDir.resolve(basename));
+        String eq = Files.readString(tmpDir.resolve(basename + "_EQ.xml"));
```

Comment:
Can be simplified as follow:
`String eq = ConversionUtil.writeCgmesProfile(network, "EQ", tmpDir);`